diff --git a/node_modules/@tonconnect/qa/dist/index.js b/node_modules/@tonconnect/qa/dist/index.js
index 2eefdf0..c0f447f 100644
--- a/node_modules/@tonconnect/qa/dist/index.js
+++ b/node_modules/@tonconnect/qa/dist/index.js
@@ -1,7 +1,7 @@
-export { Tonkeeper } from './tonkeeper/Tonkeeper';
-export { TonConnectWidget } from './TonConnectWidget';
-export { testWith, launchPersistentContext } from './test';
-export { tonkeeperFixture } from './tonkeeper/tonkeeperFixture';
-export { tonkeeperExtension } from './tonkeeper/tonkeeperExtension';
-export { getExtensionId } from './util';
+export { Tonkeeper } from './tonkeeper/Tonkeeper.js';
+export { TonConnectWidget } from './TonConnectWidget.js';
+export { testWith, launchPersistentContext } from './test.js';
+export { tonkeeperFixture } from './tonkeeper/tonkeeperFixture.js';
+export { tonkeeperExtension } from './tonkeeper/tonkeeperExtension.js';
+export { getExtensionId } from './util.js';
 //# sourceMappingURL=index.js.map
diff --git a/node_modules/@tonconnect/qa/dist/test.js b/node_modules/@tonconnect/qa/dist/test.js
index f813806..c63d491 100644
--- a/node_modules/@tonconnect/qa/dist/test.js
+++ b/node_modules/@tonconnect/qa/dist/test.js
@@ -1,6 +1,6 @@
 import { chromium } from '@playwright/test';
 import { mergeTests, test as base } from '@playwright/test';
-import { WalletExtension } from './WalletExtension';
+import { WalletExtension } from './WalletExtension.js';
 export function launchPersistentContext(extensionPath, slowMo = 0) {
     const browserArgs = [
         `--disable-extensions-except=${extensionPath}`,
diff --git a/node_modules/@tonconnect/qa/dist/tonkeeper/Tonkeeper.js b/node_modules/@tonconnect/qa/dist/tonkeeper/Tonkeeper.js
index fc10f5a..9610a54 100644
--- a/node_modules/@tonconnect/qa/dist/tonkeeper/Tonkeeper.js
+++ b/node_modules/@tonconnect/qa/dist/tonkeeper/Tonkeeper.js
@@ -1,4 +1,4 @@
-import { WalletExtension } from '../WalletExtension';
+import { WalletExtension } from '../WalletExtension.js';
 /**
  * Tonkeeper class for interacting with the Tonkeeper extension in Playwright tests
  *
diff --git a/node_modules/@tonconnect/qa/dist/tonkeeper/tonkeeperFixture.js b/node_modules/@tonconnect/qa/dist/tonkeeper/tonkeeperFixture.js
index c09e88d..2ae8a32 100644
--- a/node_modules/@tonconnect/qa/dist/tonkeeper/tonkeeperFixture.js
+++ b/node_modules/@tonconnect/qa/dist/tonkeeper/tonkeeperFixture.js
@@ -1,8 +1,8 @@
 import { test } from '@playwright/test';
-import { launchPersistentContext } from '../test';
-import { getExtensionId } from '../util';
-import { tonkeeperExtension } from './tonkeeperExtension';
-import { Tonkeeper } from './Tonkeeper';
+import { launchPersistentContext } from '../test.js';
+import { getExtensionId } from '../util.js';
+import { tonkeeperExtension } from './tonkeeperExtension.js';
+import { Tonkeeper } from './Tonkeeper.js';
 export const tonkeeperFixture = (mnemonic, slowMo = 0) => {
     return test.extend({
         context: async ({ context: _ }, use) => {
